### DISCUSSION TITLE
v3.0.79

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#9146] [General]
## What's Changed
* Add Notifications Livewire Component to Layout in Docs by @iAmKevinMcKee in https://github.com/filamentphp/filament/pull/9133
* [3.x] Add new `Markdown` helper class by @ryangjchandler in https://github.com/filamentphp/filament/pull/9140
* Added a message about the need to use an identifier. by @MarkKhromov in https://github.com/filamentphp/filament/pull/9143
* Add prompt to ask for type of widget when no type is passed in command by @iAmKevinMcKee in https://github.com/filamentphp/filament/pull/9137

## New Contributors
* @MarkKhromov made their first contribution in https://github.com/filamentphp/filament/pull/9143

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.0.78...v3.0.79

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.0.79'>v3.0.79</a>.</em>

**Comment by NazarAli** [comment]
can add sound to notification 🔊🔊🔊
